schemas/common/dataImportEventTypes.json

```diff
@@ -11,15 +11,14 @@
     "DI_INVENTORY_INSTANCE_MATCHED",
     "DI_INVENTORY_INSTANCE_NOT_MATCHED",
     "DI_INVENTORY_INSTANCE_UPDATED",
-    "DI_INVENTORY_INSTANCE_REPLACED",
```

Comment:
Plz add update for instance

Reply:
@OleksiiKuzminov We already had it before my changes: "DI_INVENTORY_INSTANCE_UPDATED".